rpc_jobs/standard_job_premerge.yml

```diff
@@ -0,0 +1,134 @@
+- project:
+    name: "gating"
+    repo_url: "https://github.com/rcbops/rpc-gating"
+    branches:
+      - "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - "default"
+    action:
+      - "deploy"
+    jobs:
+      - 'RPC_{name}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'RPC_{name}-{image}-{scenario}-{action}'
```

Comment:
This should contain the hook name or part of it, to avoid conflicts with other hooks. Also does it need the RPC prefix?

Reply:
I only used the RPC_ prefix because it was the existing convention. I'd be happy to do away with it. Also, I think by the form it's already showing what it is - but if you'd like to prefix it with 'PR-' (the trigger) then that'll work. We might want to differentiate jobs by their triggers (PR, Periodic, ?)?

Reply:
Done.